src/bls/bls.h

```diff
@@ -102,10 +107,12 @@ class CBLSWrapper
 
     void GetBuf(void* buf, size_t size) const
     {
+        assert(size >= SerSize);
```

Comment:
shouldn't this be `!=` instead?

Reply:
Why? The buffer must be of at least SerSize bytes size

Reply:
But are there any actual/potential use cases where `size > SerSize`?

Reply:
Ah you mean it should have been `==` instead? Hmm, yeah you're right, should actually always be SerSize

Reply:
ooops. yeah, `==`, sorry for confusion 🙈